Index: cacti/branches/main/lib/rrd.php
===================================================================
--- cacti/branches/main/lib/rrd.php	(revision 4870)
+++ cacti/branches/main/lib/rrd.php	(revision 4871)
@@ -95,6 +95,9 @@
 		if (sizeof($rrd_struc) == 0) {
 			session_write_close();
 			$fp = popen(read_config_option("path_rrdtool") . escape_command(" $command_line"), "r");
+			if (!$fp) {
+				unset($fp);
+			}
 		}else{
 			$i = 0;
 
@@ -129,6 +132,9 @@
 		if (sizeof($rrd_struc) == 0) {
 			session_write_close();
 			$fp = popen(read_config_option("path_rrdtool") . escape_command(" $command_line"), "rb");
+			if (!$fp) {
+				unset($fp);
+			}
 		}else{
 			$i = 0;
 
