diff -urN thold-0.41.orig/setup.php thold-0.41/setup.php
--- thold-0.41.orig/setup.php	2009-03-08 07:24:59.000000000 -0700
+++ thold-0.41/setup.php	2009-06-12 10:15:27.000000000 -0700
@@ -605,6 +605,7 @@
 		}
 		raise_message('thold_created');
 	}
+	return $action;
 }
 
 function thold_graphs_action_prepare($save) {
@@ -691,6 +692,7 @@
 			}
 		}
 	}
+	return $save;
 }
 
 function thold_graphs_action_array($action) {
