From 7a229a23c3dd3ebe40b84bb1e4b95f03e2e5b272 Mon Sep 17 00:00:00 2001
From: Konstantin vzOne Enchant <sirkonst@gmail.com>
Date: Wed, 27 May 2009 16:08:43 +0400
Subject: [PATCH 2/3] test update.php

---
 update.php |   12 ++++++++----
 1 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/update.php b/update.php
index 57b1a0f..726a038 100644
--- a/update.php
+++ b/update.php
@@ -1,13 +1,17 @@
 <?php
 /*
+
+
+!!!!!!!!!!!!!!!!TEST!!!!!!!!!!!!!!!!
+
  * CMS-remote
- * 
+ *
  * Created on 23.10.2008
  *
  * Developed by blade39
- * 
+ *
  */
- 
+
 /*KS ENGINE UPDATE SYSTEM
 
 version: 0.9
@@ -15,7 +19,7 @@ File: admin.php
 Original Code by BlaDe39 (c) 2008
 Назначение: основной файл системы администрирования, производить вывод окна логина, и отправку письма для смены пароля.
 Измениения:
-0.9 
+0.9
 	-	Переход к поддержке разнообразных кодировок и языковых файлов;
 	-	Установка кодировки UTF-8 базовой кодировкой;
 */
-- 
1.6.3

