From 06f55d589d62c7cb5a2f7e022e95b96828363181 Mon Sep 17 00:00:00 2001
From: Konstantin vzOne Enchant <sirkonst@gmail.com>
Date: Wed, 17 Jun 2009 12:39:22 +0400
Subject: [PATCH] fix last wrk with point

---
 usr/bin/gb-backup |   16 +++++++++++-----
 1 files changed, 11 insertions(+), 5 deletions(-)

diff --git a/usr/bin/gb-backup b/usr/bin/gb-backup
index 6989f90..17eb5bd 100755
--- a/usr/bin/gb-backup
+++ b/usr/bin/gb-backup
@@ -97,6 +97,7 @@ _show() {
 # поиск хеша по временной метке
 # например: 2009-06-01_13h29m -> c3cbf30e98373ad10ef080e09bcad1c79257fade
 # TODO: _point_to_hash может вернуть больше одной записи
+# TODO: удалить функцию и заменить на __get_point()
 _point_to_hash() {
 	 $GIT rev-list --grep="$1" ${ARCHIVES_BRANCH}
 }
@@ -106,10 +107,12 @@ _point_to_hash() {
 # возвращается либо точка архива если найдена либо HEAD
 __get_point() {
 	 if [ -n "$1" ] ; then
-		  # точка восстановления из тега
-		  POINT=`$GIT tag | grep "$1"`
-		  # если тег не найдет, то точка ищется в коммитах
-		  POINT=${POINT:-$(_point_to_hash "$1")}
+		# точка восстановления из тега
+		POINT=`$GIT tag | grep "$1"`
+		# если тег не найдет, то точка ищется в коммитах
+		if [ -z "$POINT" ] ; then
+			POINT=`$GIT rev-list --grep="$1" ${ARCHIVES_BRANCH}`
+		fi
 	 fi
 
 	echo "${POINT:-HEAD}"
@@ -117,8 +120,11 @@ __get_point() {
 
 # показывает последнии изменения в точке архивации
 # $1 - точка архивации (по умолчанию последняя)
+# TODO: с тега git show отображаен лишную информацию
 _last() {
-	POINT=$(__get_point)
+	shift
+	POINT=`__get_point $1`
+
 	$GIT show --name-status --pretty=format:"Изменения в архиве '%s' [%ar]" "${POINT}"
 }
 
-- 
1.6.3.1

